platforms/gke/base/use-cases/federated-learning/terraform/private_google_access/main.tf

```diff
@@ -37,7 +37,7 @@ data "google_compute_network" "main_vpc_network" {
 
```

Comment:
We might run into some issues here, since I don't believe you can have multiple zones with the same `dns_name`.